index.js

```diff
@@ -49,37 +49,40 @@ class AddressFinder extends React.Component {
     const { chosenAddress, results, error, searchTerm } = this.state;
     return (
       <div>
-        <div>Address finder:</div>
+        <label htmlFor="search">Address finder:</label>
         <div className="input_box">
           <input
+            id="search"
+            name="search"
             className="input"
             value={searchTerm}
             onChange={this.handleChange}
           />
           {searchTerm.length !== 0 && (
-            <div
+            <button
               className="clear_button"
               onClick={() => this.clearSearchValue()}
             >
-              <img src="clear.svg" />
-            </div>
+              <img src="clear.svg" alt="Clear address finder" />
+            </button>
           )}
         </div>
         {error && <div className="error">Something is wrong!</div>}
         {results.length > 0 && (
-          <div className="results">
+          <ul className="results">
```

Comment:
some screen readers read out how many items are in a list, this is useful for an address finder - if you have more than 50 results you may want to narrow down your search time! Either way, this is a list of results, so it makes sense to make it semantically a list.